packages/shared/src/components/cards/v1/TypeLabel.tsx

```diff
@@ -0,0 +1,50 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { PostType } from '../../../graphql/posts';
+
+type TypeLabelType = PostType | ReactElement | string;
+
+const typeToClassName: Record<PostType, string> = {
+  [PostType.Article]: 'text-theme-label-tertiary',
+  [PostType.Collection]: 'text-theme-color-cabbage',
+  [PostType.Freeform]: 'text-theme-label-tertiary',
+  [PostType.Share]: 'text-theme-label-tertiary',
+  [PostType.VideoYouTube]: 'text-theme-color-blueCheese',
+  [PostType.Welcome]: 'text-theme-label-tertiary',
+};
+
+const typeToLabel = {
+  [PostType.VideoYouTube]: 'Video',
+};
+
+export interface TypeLabelProps {
+  type: TypeLabelType;
+  className?: string | undefined;
+}
+
+export function TypeLabel({
+  type = PostType.Article,
+  className,
+}: TypeLabelProps): ReactElement {
+  // do not show tag label for these types
+  if (
+    type === PostType.Article ||
+    type === PostType.Freeform ||
+    type === PostType.Share ||
+    type === PostType.Welcome
```

Comment:
The code is much neater in that case, agreed. But this should be more efficient. Maybe if I create a set instead, it would be more efficient than `includes`. I know, we only have 4 elements in the array here, but still :)

Reply:
Yep, long as we won't have to add a new line for each type, I would be glad to have it 🚀